src/screens/RegisterScreen.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import styled from 'styled-components/native';
+
+import { Input, Button, Text, Wrapper } from '../components';
+
+const ScreenWrapper = styled.View`
```

Comment:
Would be good to keep the naming more verbose here. This name will be displayed in all the dev tools, and having `RegisterScreenWrapper` there is much more informative then just `ScreenWrapper`